### DISCUSSION TITLE
Firefly Ledger to Bitpanda -> Funds not received

### DISCUSSION THREAD

**Discussion by freeleux** [iotaledger/firefly#2794] [Help & Support]
I have a big issue with my Transaction from Firefly to Bitpanda i don't know why Bitpanda says they not have the amount of Iota in their adress, but i don't have it to mine too and i cans see a transaction on the explorer, so where are my Iotas...of course it some Gi...

**Comment by MiKeF05** [comment]
Check the address you have sent the IOTA to. If the transaction is confirmed and the address is the one from Bitpanda then you need to contact Bitpanda support.

**Reply by freeleux** [reply]
the adress is the good one but how can i be sure that the transaction is confirmed ? 
i have an transaction Id and this transaction said transaction of the amount XXX from my wallet adress to the BP generate adress...

**Reply by MiKeF05** [reply]
You can check it with the explorer. explorer.iota.org

**Reply by freeleux** [reply]
so if i can see this with the correct amount and adress it is ok ?


![transaction iota](https://user-images.githubusercontent.com/103255482/162440477-551e89c6-cb03-417c-801d-9449d45528df.png)

**Reply by MiKeF05** [reply]
And the transaction is confirmed? If yes you need to contact Bitpanda

**Reply by freeleux** [reply]
where do you see transaction confirmed ? on my Firtefly status is confirmed...but i can't see a status on explorer

**Reply by MiKeF05** [reply]
You need to paste the message ID. Then you should see the status

**Reply by freeleux** [reply]
the print screen was the result of my paste of the message ID  in the explorer

**Reply by MiKeF05** [reply]
If you see the following in your message Id in the explorer then the message is confirmed.

![image](https://user-images.githubusercontent.com/71659397/163585920-6049486d-1af7-47f2-9470-9ca4e151e45f.png)
